Added Flutter build for Android doc (#426)
appcircleio/appcircle-docusaurus: docs/workflows/flutter-specific-workflow-steps/flutter-build-for-android.md

### Input Variables

Each component requires specific input variables for its operation. The input variables necessary for the **Flutter Build for Android** step are:

<Screenshot url='https://cdn.appcircle.io/docs/assets/flutter-workflow-components-build_2.png'/>

| Variable Name                 | Description                                                                                           | Status    |
|-------------------------------|-------------------------------------------------------------------------------------------------------|-----------|
| `$AC_FLUTTER_PROJECT_DIR`     | Specifies the directory at the root of your Flutter project where the `pubspec.yaml` file is located. | Required  |
| `$AC_OUTPUT_TYPE`             | Defines the output type, such as `APK` or `AAB` (Android App Bundle).                                 | Required  |
| `$AC_FLUTTER_BUILD_MODE`      | Specifies the Flutter build mode. The default value is `release`.                                    | Optional  |
| `$AC_FLUTTER_BUILD_EXTRA_ARGS`| Additional custom build arguments. For instance: `--split-per-abi`.                                   | Optional  |

:::info

> If you have already completed the required variables in the **Configuration** section, there's no need to redefine them in the Workflow. For detailed information on configurations, please consult the [Build Profile Configuration Overview](/build/build-process-management/build-profile-configuration) document. The details you provide in the configuration will serve as input for the Android Build step. Kindly substitute the example information with your own details:

Reviewer: ```suggestion
If the required variables are already defined in the **Configuration** section, there is no need to redefine them in the Workflow. For more details, see the [Build Profile Configuration Overview](/build/build-process-management/build-profile-configuration). The details you provide in the configuration will serve as input for the Android Build step. Kindly substitute the example information with your details:
```

<Screenshot url='https://cdn.appcircle.io/docs/assets/flutter-workflow-components-build_3.png'/>

:::

### Output Variables

The outputs resulting from the operation of this component are as follows:

| Output Variable           | Description                     |
|---------------------------|---------------------------------|
| `$AC_APK_PATH`            | Path of the generated APK file. |        
| `$AC_AAB_PATH`            | Path of the generated AAB file. |

:::info

> The resulting files will be either APK or AAB, depending on whether you choose the `Output Type` in the project [Configuration](/build/build-process-management/build-profile-configuration).

Reviewer: ```suggestion
The resulting files will be either APK or AAB, based on the `Output Type` selected in the project [Configuration](/build/build-process-management/build-profile-configuration).
```
